pkg/apis/kubeone/types.go

```diff
@@ -1003,19 +999,22 @@ type Addon struct {
 	Delete bool `json:"delete,omitempty"`
 }
 
+type AddonRef struct {
+	// KubeOne's internal Addon
```

Comment:
This should be a valid godoc comment, maybe something like `Addon is a built-in addons mechanism`